tomcat/src/main/java/org/apache/coyote/http11/HttpHeaders.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.coyote.http11;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class HttpHeaders {
+
+    public static final String KEY_VALUE_SEPARATOR = ": ";
```

Comment:
```suggestion
    private static final String KEY_VALUE_SEPARATOR = ": ";
```
현재는 내부에서만 사용하는 것 같습니당

Reply:
추출해두고 접근제한자를 바꾸는걸 깜빡했네요! ㅠㅠ